eng/Compilers.props

```diff
@@ -6,7 +6,17 @@
     <CompilerVersion>3.6.0-2.20166.2</CompilerVersion>
   </PropertyGroup>
 
+  <PropertyGroup Condition="!($(MSBuildProjectName.EndsWith('Tests', StringComparison.OrdinalIgnoreCase)))">
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <NoWarn></NoWarn> <!-- Make sure the list is empty - by default it is not -->
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+  </PropertyGroup>
+  <PropertyGroup Condition="($(MSBuildProjectName.EndsWith('Tests', StringComparison.OrdinalIgnoreCase)))">
+    <GenerateDocumentationFile>false</GenerateDocumentationFile>
+    <NoWarn>CS1591</NoWarn> <!-- Documentation on public members missing -->
```

Comment:
Same comment here, instead of hard setting it to just this warning, let's instead do `<NoWarn>$(NoWarn);CS1591</NoWarn>` so as to not remove the warnings comming from our central infrastructure.